src/main/java/com/slimebot/games/Game.java

```diff
@@ -0,0 +1,84 @@
+package com.slimebot.games;
+
+import com.slimebot.main.Main;
+import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+public abstract class Game extends ListenerAdapter {
+
+    public final UUID uuid;
+    public long gameMaster;
+    public List<Long> player;
+    public GameStatus status;
+    public final MessageChannel channel;
```

Comment:
You should not store JDA entities. Store the channel id only and write a getChannel method that uses JDA#getChannelById